src/jvm/main/org/jetbrains/kotlinx/lincheck/IdeaPlugin.kt

```diff
@@ -435,9 +438,11 @@ private fun visualizeTrace(): Array<Any>? = runCatching {
  *
  * The Debugger uses this information to enumerate objects.
  */
-private fun createObjectToNumberMapAsArray(testObject: Any?): Array<Any> {
+@Suppress("UNUSED")
+private fun ObjectTracker.createObjectToNumberMapAsArray(testObject: Any?): Array<Any> {
     val resultArray = arrayListOf<Any>()
-    val numbersMap = if (testObject != null) enumerateObjects(testObject) else enumerateObjects()
+    // val numbersMap = if (testObject != null) enumerateReachableObjects(testObject) else enumerateAllObjects()
```

Comment:
Commented code

Reply:
Hmm, I'll give it another go to be sure. Since I tested with the verison before you rebased and updated. I remember agent going out of memory when clicking a tracepoint.

Reply:
Still works